### PR TITLE
Snippets support in Annex H, but with more structure

### PR DESCRIPTION
**Pull request by silverhook**
The main difference to https://github.com/spdx/spdx-spec/pull/718 is in structure:

- “H.2 Format” gets renamed to _“H.2 File tag format”_, but content stays the same as in existing spec.
- all snippet-level specifics are now in new _“H.3 Snippet tag format”_
- consequently, current H.3 gets bumped to _“H.4 Caveats”_
- title of the _annex is renamed_ to “Annex H SPDX File and Snippet Tags (Informative)” to reflect the changes

### CODE REVIEW

**Review by zvr — COMMENTED** [review]
Much better than #718; thanks @silverhook

File: chapters/file-tags.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
-# Annex H SPDX File Tags (Informative)
+# Annex H SPDX File and Snippet Tags (Informative)
```
[the remark this answers is not in the document]
Comment (silverhook) [reply chapters/file-tags.md]: Sounds better to me as well. As I said, I initially wanted to modify as little as possible. Fixed now.

[the remark this answers is not in the document]
Comment (silverhook) [reply chapters/file-tags.md]: Fixed, thanks for the great suggestion.

**Comment by goneall** [comment]
@kestewart We should be able to get this merged in before the release candidate.  If you're comfortable with the PR, please go ahead and merge.

**Review by kestewart — APPROVED** [review]
Extra context helps.   +1 on approving.

**Pull request merged**